just_audio/darwin/Classes/AudioPlayer.m

```diff
@@ -97,6 +97,9 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
         } else if ([@"setVolume" isEqualToString:call.method]) {
             [self setVolume:(float)[request[@"volume"] doubleValue]];
             result(@{});
+        } else if ([@"setSkipSilence" isEqualToString:call.method]) {
+            /// TODO on iOS side; Seems more involved, so someone with ObjectiveC experience might look at it.
+            result(nil);
```

Comment:
Same as for the Android implementation, this should return an encoding of the response object. It seems you are missing the corresponding method call handler for `setVolumeBoost` (although it could well be better to have that as a separate PR once we figure out how to handle audio effects in general).